ImperatorToCK3Tests/ImperatorToCK3Tests.vcxproj

```diff
@@ -296,18 +296,6 @@
     <ClInclude Include="..\ImperatorToCK3\Source\Imperator\Provinces\ProvinceName.h" />
     <ClInclude Include="..\ImperatorToCK3\Source\Imperator\Provinces\Provinces.h" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
When deleting test data files, make sure you remove the files from the test build directory (if you have the patience, I just delete the whole directory and rebuild) and make sure the tests still pass. AppVeyor also works for this, but takes even longer to see results.